web/templates/base.html

```diff
@@ -23,7 +23,7 @@
         integrity="sha384-C6RzsynM9kWDrMNeT87bh95OGNyZPhcTNXj1NW7RuBCsyN/o0jlpcV8Qyq46cDfL"
         crossorigin="anonymous"></script>
     <!-- Custom CSS -->
-    <link href="/static/styles/style.css" rel="stylesheet" />
+    <link href="/static/styles/style.css?h=8ed524cc" rel="stylesheet" />
```

Comment:
not sure what this is supposed to do...

Reply:
it's a "cache buster". This makes sure that when browsers have the style.css cached, the intended version is fetched. It does not have any impact on the server side, but the browser realized it has to fetch _this_ variant of the CSS.

It could be anything, just a distinctive character, or a timestamp; I chose for the first 8 chars of the file's sha256 checksum.

You can read more about this e.g. here, https://www.keycdn.com/support/what-is-cache-busting#3-query-strings